.vscode/settings.json

```diff
@@ -30,5 +30,36 @@
 	"stylelint.reportNeedlessDisables": true,
 	"stylelint.reportInvalidScopeDisables": true,
 	"eslint.validate": ["javascript", "javascriptreact", "typescript", "typescriptreact"],
-	"editor.wordWrap": "on"
+	"editor.wordWrap": "on",
+	"cSpell.words": [
+		"AGPL",
+		"Affero",
+		"Alltag",
+		"Benachrichtigungen",
+		"Eintrag",
+		"Eintrags",
+		"Erleichtert",
+		"Hausaufgaben",
+		"Homic",
+		"Lehrern",
+		"Sch",
+		"Schülern",
+		"Schülerportal",
+		"Verfasser",
+		"autofix",
+		"browserslist",
+		"clsx",
+		"commitlint",
+		"descriptionless",
+		"flexbugs",
+		"haptics",
+		"lern",
+		"lerportal",
+		"pcss",
+		"pnpx",
+		"preinstall",
+		"schulsolutions",
+		"stylelint",
+		"tailwindcss"
+	]
```

Comment:
Can you remove this? I'm used to saving the dictionary words under the user profile instead...